runtimes/eden/src/lib.rs

```diff
@@ -381,6 +384,65 @@ sp_api::impl_runtime_apis! {
 
 			impl frame_system_benchmarking::Config for Runtime {}
 
+			use xcm::latest::prelude::*;
+			use frame_benchmarking::BenchmarkError;
+			use pallet_xcm_benchmarks::generic::Pallet as XcmBenchmarks;
+			impl pallet_xcm_benchmarks::Config for Runtime {
+				type XcmConfig = xcm_config::XcmConfig;
+				type AccountIdConverter = xcm_config::LocationToAccountId;
+				fn valid_destination() -> Result<MultiLocation, BenchmarkError> {
+					Ok(MultiLocation::parent())
+				}
+				fn worst_case_holding() -> MultiAssets {
+				// 1 fungibles can be traded in the worst case
+					const HOLDING_FUNGIBLES: u32 = 1;
+					let fungibles_amount: u128 = 100;
+					let assets = (0..HOLDING_FUNGIBLES).map(|i| {
```

Comment:
Since we only support one asset initially I'd suggest you simplify it and avoid the range and map and for the amount go to MAX than a specific amount.